doc/13-addons.md

```diff
@@ -19,7 +19,9 @@ You need to install Graphite first, then proceed with configuring it in Icinga 2
 Use the [GraphiteWriter](14-features.md#graphite-carbon-cache-writer) feature
 for sending real-time metrics from Icinga 2 to Graphite.
 
-    # icinga2 feature enable graphite
+```
+# icinga2 feature enable graphite
```

Comment:
The leading `#` causes rendering problems when a list is put beforehand (on GH or with mkdocs, I do not remember atm). We should decide whether to drop the root shell character here.

Reply:
How about the following:

```
$ sudo icinga2 feature enable graphite
```

For example Graylog uses this to indicate that something needs to be run as root.

Reply:
I would entirely drop it, since there's also the possibility to run these commands on Windows cmd/PS.

Reply:
Agreed. I will push a new commit to update this PR. Should I remove the leading `#` only in `13-addons.md` or in the entire documentation?

Reply:
Mh. Leave it as is for now, and we'll deal with this change in a different PR. If you feel sane about this PR, feel free to merge.